app/models/event.rb

```diff
@@ -0,0 +1,33 @@
+class Event < ActiveRecord::Base
+  include Diaspora::Federated::Base
+  include Diaspora::Guid
+
+  belongs_to :status_message
+  has_many :event_participations
+
+  xml_attr :name
+  xml_attr :date
+  xml_attr :location
+
+  delegate :author, :author_id, :diaspora_handle, :public?, :subscribers, to: :status_message
+
+  validates :name, presence: true
+  validates :date, presence: true
+  validates :location, presence: true
+
+  self.include_root_in_json = false
+
+  def as_json(options={})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.